addons/hr/models/res_users.py

```diff
@@ -68,6 +68,13 @@ class User(models.Model):
     employee_type = fields.Selection(related='employee_id.employee_type', readonly=False, related_sudo=False)
 
     can_edit = fields.Boolean(compute='_compute_can_edit')
+    has_group_system = fields.Boolean(compute="_compute_has_group_system")
+
+    @api.depends_context('uid')
+    def _compute_has_group_system(self):
+        group_system = self.user_has_groups('base.group_system')
+        for user in self:
+            user.has_group_system = group_system
```

Comment:
Please rename the field, and use the existing _is_system method :)
```suggestion
    @api.depends_context('uid')
    def _compute_is_system(self):
        self.write({'is_system': self.env.user._is_system()})
```